api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -275,6 +275,13 @@ message ExternalProcessor {
   // backend stream lifetime. In this case, Envoy will eventually timeout the external processor stream according to this time limit.
   // The default value is 5000 milliseconds (5 seconds) if not specified.
   google.protobuf.Duration deferred_close_timeout = 19;
+
+  // [#not-implemented-hide:]
+  // Enables sending body to the side stream server before receives the header response from that server.
```

Comment:
```suggestion
  // Enables sending the request's body to the side stream server before receiving the header response from that server.
```

Reply:
Removed this change in the latest commit. PTAL